### PR TITLE
fix(chart): fix radar scale backdrop to be false

### PR DESCRIPTION
**Pull request by Theeraphat-Sorasetsakul**
## Description

- Fix Radar scale backdrop to be false.
- Update doughnutCenterLabel to export serveral types.

### CODE REVIEW

File: packages/elements/src/chart/plugins/doughnut-center-label.ts [hunk 1]
```diff
@@ -19,25 +19,25 @@ interface Selectable {
...
+export interface CenterLabelConfig {
   defaultText: CenterLabel[];
   selected: Selected;
   onRenderLabel(chart: ChartJS, chartItems: ActiveElement[]): CenterLabel[];
 }
 
 declare module 'chart.js' {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  interface PluginOptionsByType<TType extends ChartType> {
+  interface PluginOptionsByType<TType extends ChartType = ChartType> {
```
Comment (wattachai-lseg) [inline packages/elements/src/chart/plugins/doughnut-center-label.ts:40]: do we need this default value?

**Review by wattachai-lseg — COMMENTED** [review]
1 thing.

File: packages/elements/src/chart/plugins/doughnut-center-label.ts (hunk 1 above)
Comment (Theeraphat-Sorasetsakul) [reply packages/elements/src/chart/plugins/doughnut-center-label.ts:40]: I just update to follow suggestion in Chart.js doc. https://www.chartjs.org/docs/latest/developers/plugins.html#typescript-typings

Comment (Theeraphat-Sorasetsakul) [reply packages/elements/src/chart/plugins/doughnut-center-label.ts:40]: Updated to both interfaces.

**Review by wattachai-lseg — APPROVED** [review]

**Pull request merged**